89.md

```diff
@@ -0,0 +1,107 @@
+NIP-89
+======
+
+Recommended Application Handlers
+--------------------------------
+
+`draft` `optional` `author:pablof7z`
+
+This NIP describes `kind:31989` and `kind:31990`: a way for unknown event-kinds application-handlers to be discovered.
+
+## Rationale
+Nostr's discoverability and transparent event interaction is one of its most interesting/novel mechanics. This NIP seeks to provide the glue for cross-`kind` client interaction.
```

Comment:
This NIP provides a simple way for clients to discover applications that handle events of a specific `kind` to ensure smooth cross-client and cross-`kind` interactions.